codec-http2/src/main/java/io/netty/handler/codec/http2/WeightedFairQueueByteDistributor.java

```diff
@@ -338,5 +381,33 @@ public int priorityQueueIndex() {
         public void priorityQueueIndex(int i) {
             priorityQueueIndex = i;
         }
+
+        @Override
+        public String toString() {
+            StringBuilder sb = InternalThreadLocalMap.get().stringBuilder();
```

Comment:
If we expect to call this from outside the EventLoop please just create a new StringBuilder.
See https://github.com/netty/netty/pull/5998